arrow/src/main/java/org/apache/iceberg/arrow/vectorized/VectorizedReaderBuilder.java

```diff
@@ -120,21 +149,24 @@ public VectorizedReader<?> struct(
   @Override
   public VectorizedReader<?> primitive(
       org.apache.iceberg.types.Type.PrimitiveType expected, PrimitiveType primitive) {
-
```

Comment:
Please avoid formatting functions unrelated to the PR

Reply:
Sorry, I should have caught this. I'll revert it.

Reply:
Fixed.